app/backend/approaches/retrievethenread.py

```diff
@@ -165,9 +204,18 @@ async def run(
                 ),
             ],
         }
-
         completion = {}
-        completion["message"] = chat_completion["choices"][0]["message"]
+        if self.use_hugging_face:
+            completion["message"] = {
+                "content": chat_completion[0]['generated_text'],
```

Comment:
Let's default to double quotes for strings everywhere!
```suggestion
                "content": chat_completion[0]["generated_text"],
```